dev/com.ibm.ws.ui.tool.explore/bnd.bnd

```diff
@@ -12,6 +12,7 @@
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/bundle.props
 bVersion=1.0
+-noparallel: ui
```

Comment:
What part of ui projects do not work with parallel?

Reply:
I've reverted these UI changes in hopes of finding a fix that allows them to still be run in parallel. I also don't have the issue trace, so I am hoping to recreate the problem.

Reply:
Error I've been seeing with UI projects running in parallel is:

```
Execution failed for task ':com.ibm.ws.ui.tool.explore:dojo-release-execute'.
> The following error occurred while executing this line:
  /Users/erfritz/workspace/websphere/open-liberty/dev/ant_build.js/public_imports/internal_imports/dojo.xml:326: /Users/erfritz/workspace/websphere/open-liberty/dev/ant_build.js/public_imports/internal_imports/dojo.xml:384: Result of nodejs build: ${nodejsResult}.  Failed to build dojo with nodejs.  Not building with rhino since this is a local build.  If you're seeing this and haven't made changes to a com.ibm.ws.ui* project, then there's likely something in your environment that's prevent nodejs from running properly.  Try building with Rhino instead.  To build with rhino, set the ant property "dojo.build.engine.force.rhino" to "true".  Check build/dojo/build-report.txt for errors.
```

build/dojo/build-report.txt does not exist, so I'll need to get more logging printed into the console.

Reply:
The build output suggests to use Rhino instead. I've tried Rhino and it doesn't encounter this issue (yet). I'll contact the UI team to see if they have thoughts on how to debug this and how to move forward.